sound/pci/hda/hda_intel.c

```diff
@@ -2495,6 +2495,8 @@ static const struct pci_device_id azx_ids[] = {
 	{ PCI_DEVICE_DATA(INTEL, HDA_ARL_S, AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE) },
 	/* Arrow Lake */
 	{ PCI_DEVICE_DATA(INTEL, HDA_ARL, AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE) },
+	/* Panther Lake */
+	{ PCI_DEVICE_DATA(INTEL, HDA_PTL, AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE) },
```

Comment:
Should we verify that PTL needs AZX_DCAPS_INTEL_LNL (iow, HDA PIO commands)?

Reply:
yes, also copy-paste before your LNL PR, thanks!

Reply:
done